### ISSUE TITLE
RMB sentence specifies invalid keys

### ISSUE THREAD

**Issue by panaaj** [SignalK/signalk-to-nmea0183#60]
RMB.js specifies the following keys..
```
sentence: 'RMB',
    title: 'RMB - Heading and distance to waypoint',
    keys: [
      'navigation.courseRhumbline.crossTrackError',
      'resources.waypoints.next.position.latitude',
      'resources.waypoints.next.position.longitude',
      'navigation.courseRhumbline.nextPoint.distance',
      'navigation.courseRhumbline.bearingTrue'
    ],
...

```
the keys `resources.waypoints.next.position.latitude` and `resources.waypoints.next.position.longitude` are not part of the spec.

I would suggest  that these keys should be replaced with `navigation.courseRhumbline.nextPoint.position`.

Also the use of `courseRhumbline` vs `courseGreatCircle` should be reviewed for consistency across sentences (`APB` uses `courseGreatCircle`). Additionally `signalk-derived-data` plugin populates `courseGreatCircle` paths.

**Comment by dpazz** [comment]
I agree with you because the "resources..." starting path is unavailable in SignalK data browser and the RMB sentence is NEVER generated by the plugin. I'm trying to drive autopilot directly via SignalK. The XTE is correctly generated but not the APB or RMB, because of those unavailable "resources.." keys. In my case the APB is sent by an external nav app (AvNav) via an UDP writer to SignalK ad is correctly parsed and stored in SignalK data format in various keys. The next waypoint info is stored in the key "navigation.courseRhumbline.nextPoint" with two subkeys 

{
  "longitude": 10.21665,
  "latitude": 43.76265
}

There's something to fix here in my humble opinion....

**Comment by msallin** [comment]
@panaaj I think this is now fixed.

**Comment by msallin** [comment]
Fixed.
